pywps/__init__.py

```diff
@@ -171,12 +171,12 @@ class Pywps:
     versions=[DEFAULT_VERSION]
     UUID = None
 
-    def __init__(self, method=METHOD_GET, configFiles=None):
+    def __init__(self, environ, configFiles=None):
```

Comment:
give an entry for the request environment
Maybe this breaks non-wsgi applications. Maybe something along the lines of:
def __init__(self, environ = {"REQUEST_METHOD":"METHOD_GET"}, configFiles=None):
is better.